src/plugins/vis_type_table_new/opensearch_dashboards.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "visTypeTableNew",
+  "version": "opensearchDashboards",
```

Comment:
since it is a refactor from old, should we just use `"version": "opensearchDashboards"`?

Reply:
Interesting, I have never seen version being used in this way. I actually dont have a lot of context here. I left this section untouched from the autogenerated template when i created the VisBuilder plugin [[Ref](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/packages/osd-plugin-generator/template/opensearch_dashboards.json.ejs)]

That being said, this is more relevant as far as i can tell for external plugins since its used to generate manifest files and check for version compatibility. e.g. https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/packages/osd-plugin-helpers/src/tasks/update_versions.ts#L19

I have a hunch that having this value has something to do with whether its an internal or external plugin since internal plugins do not need matching versions with dashboards versions. I'd keep it as is for now since it works for other plugins and create a followup task to research on the implications of this value.

Reply:
Follow up issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2751
I copied both of the feedbacks in the above issue.